client/web/web.go

```diff
@@ -283,6 +288,11 @@ func (s *Server) serve(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if debugUsermetricsEndpoint() && strings.HasPrefix(r.URL.Path, "/metrics") {
```

Comment:
I think this won't work if the web client is started as a separate process via `tailscale web`, and we need to expose the metric endpoint via localapi, with the web client calling a localapi endpoint.

Reply:
oh yes, obviously, I did an implementation of this yesterday, ptal, I might have been a bit creating reusing the proxy func, might want to split it to a new func for metrics specifically.